libs/entitlement/tools/jdk-api-extractor/build.gradle

```diff
@@ -0,0 +1,68 @@
+plugins {
+  id 'application'
+}
+
+apply plugin: 'elasticsearch.build'
```

Comment:
FWIW, this isn't for "production" use so we probably don't need to apply this unless we think forbidden apis, and checkstyle and all that provides benefit.